internal/pkg/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -79,7 +79,10 @@ COPY build/_output/bin/[[.ProjectName]] ${OPERATOR}
 COPY bin /usr/local/bin
 COPY library/k8s_status.py /usr/share/ansible/openshift/
 
-RUN /usr/local/bin/user_setup
+RUN chmod -R g+rwx /usr/share/ansible/openshift/
```

Comment:
Nit: I don't _think_ we need execute permission here (@fabianvf, correct me if I'm wrong)

```suggestion
RUN chmod -R g+rw /usr/share/ansible/openshift/
```